app/static/js/events/wizard/vue-app.js

```diff
@@ -350,7 +354,9 @@ function shouldDisableMove($this) {
         $this.event.start_time_time.trim() === '' ||
         $this.event.start_time_date.trim() === '' ||
         $this.event.end_time_time.trim() ==='' ||
-        $this.event.end_time_date.trim() === ''
+        $this.event.end_time_date.trim() === '' ||
+        $this.ticket.price === '' ||
```

Comment:
This is now how you're supposed to check. Each event will have multiple tickets. You'll have to loop through `this.event.tickets`